src/index.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable quotes */
```

Comment:
- [ ] Please kindly do not disable the linters, fix linters help us to write a clean and professional codebase 🙃 😉